config/initializers/audit.rb

```diff
@@ -5,3 +5,10 @@
 
 Audit = Audited.audit_class
 Audit.send(:include, AuditExtensions)
+
+# if this is a migration rake task we disable auditing in particular models
+if Foreman.in_rake?('db:migrate')
+  [ Host::Managed, Hostgroup, ProvisioningTemplate, Ptable, User ].each do |klass|
```

Comment:
Hm yeah, and the extension I pointed to is actually the Audit model and not the concern that's loaded into AR::Base.  That'd be Audited::Auditor.

So you're back to either adding it into every model, or an `alias_method_chain` around `audited` to set it back to false after?